bindgen-cli/options.rs

```diff
@@ -589,7 +590,11 @@ where
 
     let mut builder = builder();
 
-    builder = builder.header(header);
+    if let Some(header) = header {
+        builder = builder.header(header);
+    } else {
+        return Err(IoError::new(IoErrorKind::Other, "Header not found"));
```

Comment:
I would not create aliases, as using the actual path is readable enough. Just a thought... feel free to ignore.

```suggestion
        return Err(io::Error::new(io::ErrorKind::Other, "Header not found"));
```

Reply:
Yeah I don't mind changing it to this.

5607d8586531650afad4852fa2bb3c4612f3fe66